docs/using-jda/interactions.md

```diff
@@ -96,64 +97,116 @@ When you use `deferReply` the first message sent to this webhook will act identi
-
-Each button can be enabled or disabled, have a specific style, label, and emoji:
+# Component Interactions
 
-![Example Button Styles](https://raw.githubusercontent.com/DV8FromTheWorld/JDA/52377f69d1f3bfba909c51a449ac6b258f606956/assets/wiki/interactions/ButtonExamples.png)
-
-To add such buttons to a message you can use up to 5 ActionRows.
+To add components to a message you can use up to 5 ActionRows.
 
 You can add multiple ActionRows with either `setActionRows` or `addActionRows` (depending on whether you are sending or editing a message).
-For the common case of a single ActionRow you can also use `setActionRow(Button...)` or `addActionRow(Button...)`.
-
-### Handling ButtonClickEvent
+For the common case of a single ActionRow you can also use `setActionRow(Component...)` or `addActionRow(Component...)`.
 
-To properly use an interactive Button, meaning any non-link button, you need to use the **Component ID** (aka **Custom ID**, **Button ID**).
-Each non-link button requires such an ID in order to be used and this ID can also be used to then *identify* which button was pressed by the user.
+Each ActionRow can hold up to a certain amount of components:
+- 5 Buttons
+- 1 Selection Menu (Dropdown)
 
-When a user presses one of these buttons, you will receive a `ButtonClickEvent` for the respective interaction. This event provides the configured ID with `getComponentId()`.
-
-These button interactions, or component interactions, offer 4 response types:
+These component interactions offer 4 response types:
 
 - Reply
 - Deferred Reply
 - Edit Message
 - Deferred Edit Message
 
-The reply and deferred reply responses are identical to the Slash-Commands response types. However, these new edit response types are used to update the existing message the button is attached to. If you simply want to acknowledge that the button was pressed, you can simply do a `deferEdit()` without any further updates, which will prevent the interaction from failing on the user side.
+The reply and deferred reply responses are identical to the Slash-Commands response types. However, these new edit response types are used to update the existing message the component is attached to. If you simply want to acknowledge that the component was successfully interacted with, you can simply do a `deferEdit()` without any further updates, which will prevent the interaction from failing on the user side.
+
+To properly use an interactive component, you need to use the **Component ID** (aka **Custom ID**).
+This ID can also be used to then *identify* which component was pressed by the user.
+
+Such Component ID is provided by `getComponentId()` on every Component Interaction.
```

Comment:
It might be worth mentioning these must be unique across all components in a message